src/GafferSceneModule/EditScopeAlgoBinding.cpp

```diff
@@ -269,6 +269,13 @@ bool renameRenderPassWrapper( Gaffer::EditScope &scope, const std::string &oldNa
 	return EditScopeAlgo::renameRenderPass( &scope, oldName, newName );
 }
 
+std::string renameRenderPassNonEditableReasonWrapper( Gaffer::EditScope &scope, const std::string &newName )
+{
+	IECorePython::ScopedGILRelease gilRelease;
+	std::optional<std::string> result = EditScopeAlgo::renameRenderPassNonEditableReason( &scope, newName );
+	return result ? result.value() : "";
```

Comment:
Although it'd be reasonable to use "" to mean "editable", since we're using `optional` in C++, I think the right return value for the editable case is `None` in Python.

Reply:
Alternatively we could drop `optional` on the C++ side and use "" as our editable value on both sides.